Ref/Top/RefPackets.xml

```diff
@@ -72,12 +72,7 @@
         <channel name="systemResources.NON_VOLATILE_FREE"/>
     </packet>
 
-    <packet name="SystemRes2" id="6" level="2">
-        <channel name="systemResources.FRAMEWORK_VERSION"/>
-        <channel name="systemResources.PROJECT_VERSION"/>
-    </packet>
-
-    <packet name="SystemRes3" id="7" level="2">
+    <packet name="SystemRes3" id="6" level="2">
```

Comment:
Is it okay that I changed id for SystemRes3 from 7 to 6?

Reply:
As long as it is unique (which it is) it should not matter. We could also rename `SystemRes3` to `SystemRes2` since there is no `SystemRes2` anymore, but not really needed